entries/s/socalgas.com.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Southern California Gas Company": {
+    "domain": "socalgas.com",
+    "img": "site.com.png",
+    "tfa": ["sms", "totp"],
+    "documentation": "https://www.socalgas.com/help-center/2-factor-authentication",
+    "regions": ["us"],
+    "categories": ["utilities"]
```

Comment:
Thanks for the pull request @johntheholman.
Could you please use the code style that the rest of the JSON files use? Where `{` and `[` are followed by new lines and `]` `}` are prefixed by new lines.

Reply:
I will update this. Not sure why the config is not working. Looking into that.